### PR TITLE
System tray icon

### LINKED ISSUE #219: System Tray Icon
Can you add an icon in the system tray and allow museeks to run in the background.

### PR DESCRIPTION
**Pull request by KeitIG**
Fix #219

### CODE REVIEW

File: src/js/actions/AppActions.js [hunk 1]
```diff
@@ -97,7 +103,16 @@ const AppActions = {
         },
 
         close: function() {
-            app.browserWindows.main.close();
+
+            if(app.config.get('useTray')) {
```
Comment (YurySolovyov) [inline src/js/actions/AppActions.js]: Weird key name. How about `minimizeToTray` ?

File: src/js/actions/SettingsActions.js [hunk 2]
```diff
@@ -122,6 +122,16 @@ export default {
         AppActions.app.restart();
     },
 
+    toggleTray: function() {
```
Comment (YurySolovyov) [inline src/js/actions/SettingsActions.js]: I wonder why not use `checked` property instead of toggling.

File: src/main-process/tray.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    constructor(win, icon) {
+
+        this.tray = null;
+        this.trayIcon = icon;
+        this.win = win;
+    }
+
+    bindEvents() {
+
+        const self = this;
```
Comment (YurySolovyov) [inline src/main-process/tray.js]: I think with arrow functions this is unnecessary

File: src/js/actions/AppActions.js (hunk 1 above)
Comment (KeitIG) [inline src/js/actions/AppActions.js]: Fair

File: src/js/actions/SettingsActions.js (hunk 2 above)
Comment (KeitIG) [inline src/js/actions/SettingsActions.js]: That's a fair concern, it's just the other actions are made that way. I'll open an issue for that.

File: src/main-process/tray.js (hunk 3 above)
Comment (KeitIG) [inline src/main-process/tray.js]: True

**Comment by KeitIG** [comment]
About the checked property, I'm doing it right now, I'll push it in this PR

**Pull request merged**